src/components/Button/index.tsx

```diff
@@ -287,16 +286,16 @@ type StyledButtonProps = {
   size: ButtonSize
   tooltip?: string
   tooltipBaseId?: string
-  type?: 'button' | 'reset' | 'submit'
   variant: ButtonVariant
-  onFocus?: FocusEventHandler
-  onMouseEnter?: MouseEventHandler
-} & BoxProps &
-  ButtonHTMLAttributes<HTMLButtonElement>
+} & ButtonHTMLAttributes<HTMLButtonElement> &
+  Pick<
+    AnchorHTMLAttributes<HTMLAnchorElement>,
+    'href' | 'target' | 'download' | 'rel'
+  >
 
-const StyledButton = styled(Box, {
+const StyledButton = styled('button', {
   shouldForwardProp: prop =>
-    !['action', 'variant', 'extend', 'icon', 'download'].includes(prop),
+    !['action', 'variant', 'extend', 'icon', 'download', 'as'].includes(prop),
```

Comment:
why? This should be forwarded to emotion no ?

Reply:
otherwise it goes to html & the `as` isn't applied, feel free to try without on "Download" story but without `a` marker don't replace `button` marker